File: ITP1/4_A.js
function Main(input){
  const [a,b] = input.split(' ').map(Number)
	const ans = Math.floor(a/b)
	const ans2 = a % b
	const ans3 = a / b
  console.log(ans + ' ' + ans2 + ' ' + ans3.toFixed(10));
}

Main(require("fs").readFileSync("/dev/stdin", "utf8"));

File: ALDS1/4_A.py
n = int(input())
S = list(map(int, input().split()))
q = int(input())
T = list(map(int, input().split()))

c = 0
for i in range(q):
    for j in range(n):
        if T[i] == S[j]:
            c += 1
            break

print(c)
